cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -428,6 +429,16 @@ In order to configure CDAP Master for Kerberos authentication:
       process. The string "_HOST" will be substituted with the local hostname.</description>
     </property>
 
+- The ``/cdap`` directory needs to be owned by the ``<cdap-principal>``; you can set
+  that by running the following command as the ``hbase`` user (substituting for
+  ``<cdap-principal>``)::
+  
+    hadoop fs -mkdir /cdap && hadoop fs -chown <cdap-principal> /cdap
+    
+- When running on a secure HBase cluster, as the ``hbase`` user, issue the command::
+
+    echo "grant 'cdap', 'ACRW'" | hbase shell
```

Comment:
Sorry should have caught this before, we should replace 'cdap' here with '<cdap-principal>' and repeat instructions to substitute the cdap prinicpal name.